client/src/Try/App.purs

```diff
@@ -0,0 +1,70 @@
+module Try.App
+  ( AppT(..)
+  , Error(..)
+  , ParAppT
+  , runAppT
+  ) where
+
+import Prelude
+
+import Control.Monad.Error.Class (class MonadThrow)
+import Control.Monad.Except (ExceptT(..))
+import Control.Monad.Except as ExceptT
+import Control.Parallel as Parallel
+import Control.Parallel.Class (class Parallel)
+import Data.Array.NonEmpty (NonEmptyArray)
+import Data.Either as Either
+import Data.Functor.Compose (Compose(..))
+import Data.Newtype as Newtype
+import Data.Validation.Semigroup (V)
+import Data.Validation.Semigroup as V
+import Effect.Aff.Class (class MonadAff)
+import Effect.Class (class MonadEffect)
+
+data Error
+  = FetchError String
+  | FFIErrors (NonEmptyArray String)
```

Comment:
Realizing that I probably want to capture all the errors here, so `CompilerErrors` as well, though I'm not sure why those are currently tracked separately from the error type of `ExceptT`. For example https://github.com/purescript/trypurescript/blob/0bd14e29e67b43d9252bdaab1d4acba3a23ffa30/client/src/Try/API.purs#L101 seems like it has three error channels going on: there are two `String`s (in `ExceptT String` and `Either String`) and the possibility of `CompileResult` to be `CompileFailed`.